### DISCUSSION TITLE
(ANSWERED) Right Ascension and Precession.

### DISCUSSION THREAD

**Discussion by Flapnuggets** [skyfielders/python-skyfield#619] [Q&A]
Hi, hopefully you guys can help me out.  I am building a Nakshatra to Right Ascension convertor using de441 data to date ancient Hindu texts.  Luckily, the Nakshatras are fixed in position.  From my research I found that Right Ascension's zero point is always the Vernal equinox, and may be impacted by Precession.

I have a couple of questions:

1) Is Right Ascension impacted by precession?  If so, would setting the epoch return more accurate results for planetary positions?  
2) How is epoch set in Skyfield?  In the old pyephem, there was an option to set the epoch in compute:

>>> polaris.compute(epoch='2100')
>>> print(polaris.a_dec)
89:32:26.1

I appreciate in advance!  I love the module :).

**Comment by brandon-rhodes** [comment]
Extended discussion of this question happened over in the accompanying issue #628 but here’s a summary for folks who find this Discussion:

1. Yes, RA is affected by precession and nutation, so you need to specify an equinox date if you don't want simple J2000 coordinates.
2. You can specify the date to which you want the coordinates precessed by providing that date to the `.radec()` call, like this: `.radec(equinox_date)`.
3. But if you are trying to parse ancient astrology, then instead of looking at Right Ascension and modern constellation boundaries, you probably want to look at ecliptic longitude, which measures distance along the ecliptic — which is what most ancient observers of the sky used to measure planetary positions. You will, in fact, need to research exactly how the culture did its ancient observations, and use their exact definitions for things like “house” or “nakshatra”. You might even need to compensate for how their system of observations applied those names or categories, as they are unlikely to have been able to measure ecliptic longitude with anything close to modern precision.
